src/main/java/Event.java

```diff
@@ -0,0 +1,21 @@
+public class Event extends Task {
+    protected String from;
```

Comment:
Could consider naming it "at" for ease of understanding that event is on the day itself